packages/rancher-alerting-drivers/charts/app-readme.md

```diff
@@ -0,0 +1,9 @@
+# Rancher Alerting Drivers
+
+Rancher-alerting-drivers is a tool for Alertmanger to send alerts to [non-naively-supported webhooks](https://prometheus.io/docs/operating/integrations/#alertmanager-webhook-receiver).
+
+Currently, it supports sending alerts to the following:
+- Microsoft Teams, based on [Prom2teams](https://github.com/idealista/prom2teams)
+- Alibaba Cloud SMS, based on [Sachet](https://github.com/messagebird/sachet)
+
+After installing the rancher-alerting-manager app, please refer the documentation of the sub-chart for configuration and usage.
```

Comment:
We should also add a README.md for dev side of things. See other charts for examples.

Reply:
the file is added.